src/index.spec.js

```diff
@@ -28,6 +28,14 @@ const data = deepFreeze({
       username: 'Administrator',
       password: 'pas:sw@ord'
     }
+  },
+  NFS: {
+    string: 'nfs://192.168.100.225:media/nfs',
```

Comment:
Should the parser be able to "fix" this for previous created remotes, and I remove the extra slash on server side with remote parser 3 a s a dependency ?